lib/Parser/Parse.cpp

```diff
@@ -2436,19 +2436,26 @@ bool Parser::IsImportOrExportStatementValidHere()
 }
 
 template<bool buildAST>
-ParseNodePtr Parser::ParseImportDeclaration()
+ParseNodePtr Parser::ParseImport()
 {
     Assert(m_scriptContext->GetConfig()->IsES6ModuleEnabled());
     Assert(m_token.tk == tkIMPORT);
 
+    m_pscan->Scan();
+
+    // import()
+    if (m_token.tk == tkLParen)
+    {
+        ParseNodePtr specifier = ParseTerm<buildAST>();
+        return CreateCallNode(knopCall, CreateNodeWithScanner<knopImport>(), specifier);
```

Comment:
>CreateCallNode [](start = 15, length = 14)

Since we parse this like a call, I would like to make sure we have sufficient coverage for parsing cases where import() could be mistaken for a function call, e.g. with multiple parameters, spread, etc.